src/EntityFramework.Sqlite/SqliteSqlGenerator.cs

```
@@ -10,6 +10,8 @@ namespace Microsoft.Data.Entity.Sqlite
 {
     public class SqliteSqlGenerator : SqlGenerator
     {
+        public override string DelimitIdentifier(string tableName, string schemaName) => base.DelimitIdentifier(tableName);
```

Hmm, I think it's an error to ever call this with a schema name. Ignoring this might be hiding other errors. Asking for the SQLite-specific schema name in metadata will always return null. If callers are getting the schema some other way, it's wrong.